### PR TITLE
Add Class and Style parameters to BitLoading Components (#6392)

### LINKED ISSUE #6392: Cannot give `BitLoading` Components styles

### PR DESCRIPTION
**Pull request by mhrastegari**
Closes #6392

### CODE REVIEW

File: src/BlazorUI/Bit.BlazorUI/Components/Loading/BitDualRingLoading.razor [hunk 1]
```diff
@@ -2,13 +2,13 @@
 @inherits BitLoadingComponentBase
 
 <style>
-    .blazor-loading-dual-ring {
+    .bit-ldn-DuaRing {
```
Comment (msynk) [inline src/BlazorUI/Bit.BlazorUI/Components/Loading/BitDualRingLoading.razor:5]: plz fix the typo: `DuaRing`  =>  `DualRing`

File: src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Components/ComponentDemo.razor [hunk 2]
```diff
@@ -33,14 +33,17 @@
             <ComponentParametersTable Name="BitInputBase" Parameters="_inputBaseParameters" />
         }
 
-        <ComponentParametersTable Name="BitComponentBase" Parameters="_componentBaseParameters" />
+        @if (ComponentBaseParameters is not null)
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Components/ComponentDemo.razor:36]: good point 👌
but this change is not related to this issue, please roll it back.

File: src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Components/ComponentDemo.razor.cs [hunk 3]
```diff
@@ -10,10 +10,7 @@ public partial class ComponentDemo
     [Parameter] public List<ComponentSubClass> ComponentSubClasses { get; set; } = new();
     [Parameter] public List<ComponentSubEnum> ComponentSubEnums { get; set; } = new();
     [Parameter] public List<ComponentParameter> ComponentPublicMembers { get; set; } = new();
-    
-
-
-    private readonly List<ComponentParameter> _componentBaseParameters = new()
+    [Parameter] public List<ComponentParameter>? ComponentBaseParameters { get; set; } = new()
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Components/ComponentDemo.razor.cs:13]: this change is not related to this issue, please roll it back.

File: src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Loading/BitLoadingDemo.razor [hunk 4]
```diff
@@ -5,6 +5,7 @@
             Description="loading component of the bit BlazorUI components" />
 
 <ComponentDemo ComponentName="Loading"
+               ComponentBaseParameters="null"
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Loading/BitLoadingDemo.razor:8]: this change is not related to this issue, please roll it back.

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Loading/BitLoadingDemo.razor (hunk 4 above)
Comment (mhrastegari) [reply src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Loading/BitLoadingDemo.razor:8]: Since we always update demo when we adding new parameters, I did these changes! and those totally related to BitLoading demo to show correct parameters :)

 am I wrong?

Comment (msynk) [reply src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Loading/BitLoadingDemo.razor:8]: yes, you are wrong

**Review by msynk — APPROVED** [review]
